app/views/admin/page/index.html.erb

```diff
@@ -3,5 +3,7 @@
     <h1 class="govuk-heading-xl">
       Admin
     </h1>
+
+    <%= link_to 'Download claims', admin_claims_path(format: :csv) %>
```

Comment:
```suggestion
    <%= link_to 'Download claims', admin_claims_path(format: :csv), class: "govuk-button" %>
```

Reply:
I did consider this, but `govuk-button` is rarely used for links (apart from on start pages), so I decided against it. I guess it's not a public facing page, so is probably OK here